src/components/ResortsData.js

```diff
@@ -37,7 +37,8 @@ class ResortsData extends Component{
           }
         })
         .then(({data, headers}) => {
-          this.setState({ data: data,  pageCount: Math.ceil(headers.get('x-total-count')/ this.state.perPage), isLoading: false })
+          this.setState(
+           {data, pageCount: Math.ceil(headers.get('x-total-count')/this.state.perPage), isLoading: false})
```

Comment:
what I meant, in such cases to shorten the lines split them like that.

```
this.setState({
  data,
  pageCount: Math.ceil(headers.get('x-total-count')/this.state.perPage),
  isLoading: true,
})
```

Reply:
OK :)